pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -65,6 +65,9 @@ type CronWorkflowSpec struct {
 	StopStrategy *StopStrategy `json:"stopStrategy,omitempty" protobuf:"bytes,10,opt,name=stopStrategy"`
 	// Schedules is a list of schedules to run the Workflow in Cron format
 	Schedules []string `json:"schedules,omitempty" protobuf:"bytes,11,opt,name=schedules"`
+	// v3.6 and after: When clause can be used to determine a run should or shouldn't be scheduled.
+	// This new When clause allows for the full expressivity of expr-lang.
```

Comment:
```suggestion
```

This is similarly not necessary. You should just say it uses an expression, which is defined in the "Variables" page and linked to from various places as well